src/clusterfuzz/_internal/build_management/build_manager.py

```diff
@@ -612,7 +611,7 @@ def _get_fuzz_targets_from_archive(self, reader):
     # Import here as this path is not available in App Engine context.
     from clusterfuzz._internal.bot.fuzzers import utils as fuzzer_utils
 
-    for archive_file in reader.list_files():
+    for archive_file in reader.list_members():
       if fuzzer_utils.is_fuzz_target_local(
           archive_file.filename, reader.try_open(archive_file.filename)):
```

Comment:
3 references to `archive_file.filename` should be changed to `archive_file.name`. The fact that this works means we must be missing a test?

Reply:
Well... those are hard to test, because we must mock what `list_members()` returns and ensure the members are "called". I think it is not necessary, mainly because this is a detail of the implementation of `build_manager.py`.
I think this is rather something to be caught with python type-checking rather than an actual test.

However, we do test what exactly `list_members()` does in `archive_test.py`, and we do ensure that it returns what's expected.

Reply:
I don't mean to suggest we should have an additional test for `ArchiveMemberInfo`, rather that this `build_manager` code seems unsufficiently tested. I can't figure out exactly what is supposed to be testing it, but in general we mock out `archive.open()` in `build_manager_test.py`, so I don't think the `list_members()` call will ever return a non-empty list, and we'll never hit this code. We might also be mocking out some function higher in the stack, in which case we'll never even execute this function.

Anyway, if you can't easily figure it out, let's leave it at that.